docs/REQUIREMENTS.md

```diff
@@ -7,8 +7,22 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "S
 
 ## Definitions
 
-- _**Term**: Definition._
+- **Namespace**: A namespace allows a container to have its own isolated
+  instance of a system resource. When using FreeBSD jails, this can be network,
+  UTS or IPC.
 
 ## User Stories
 
-1. As a _role_, I want to _task_.
+1. As a user, I want control over which namespaces are private to a container
+   and which are shared with the host.
+2. As a user, I want to be able to group containers into pods which share
```

Comment:
```suggestion
2. As a user, I want to be able to define groups of containers which share
```

nit: pod isn't an OCI term, but is a Kubernetes concept.  Slightly different level of abstraction.